app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -558,8 +558,8 @@ private void setupBottomBar(View rootView) {
             bottomBarItemAdapter.setNightMode(isNightMode);
             return Unit.INSTANCE;
         });
-        LiveDataExtensionKt.nonNullObserve(chromeViewModel.getTabCount(), this, changedEvent -> {
-            bottomBarItemAdapter.setTabCount(changedEvent.getCount(), changedEvent.getWithAnimation());
+        LiveDataExtensionKt.nonNullObserve(chromeViewModel.getTabCount(), this, count -> {
+            bottomBarItemAdapter.setTabCount(count, true);
```

Comment:
I just check the previous implementation, only the one in `onSessionCountChanged` should show tab count change animation.